### ISSUE TITLE
Multisig Acc redundant-confusing string "Accounts" button [Enhancement]

### ISSUE THREAD

**Issue by fiazk** [NewEconomyMovement/NEMAndroidApp#216]
The string "Accounts" button in the multisig acc seems redundant (the account name is already displayed at the top) and confusing since it's displayed above the "Send" button and not where the account name is displayed at the top. Perhaps moving the button to the top right next to the account name and or changing the string for "Switch Acc" would make more sense
![photo_2016-06-15_13-35-14](https://cloud.githubusercontent.com/assets/17099341/16078229/fdf33412-32fd-11e6-906a-7cd95885306e.jpg)

NEM Address: NDAZM2-KCD2XB-4Y7XMM-J5UUZ3-2ZCBHE-MHRHF4-3U3W
Mobile: Samsung Galaxy S3
NEM version 1.0.49

**Comment by fyahfox** [comment]
Hm, it is definitely not a good solution like it is, but I would still like to see more ideas until we try a fix. This is a difficult UI problem. Not much real estate on phones... :/

**Comment by fiazk** [comment]
@fyahfox do you find the "Accounts" string correct to be right next to amount and it's meaning. I.e. "Send" it's a action, I know what will happen if I press that button but with "accounts" I don't I think it should be at least something like "Switch Acc" or "Select Acc"

**Comment by fyahfox** [comment]
No, I don't like the current solution. But we need a good idea before we implement a fix.
I personally don't have one till now.

**Comment by jabo38** [comment]
Right now we can't pay a bounty for major UI suggestions because that costs extra and UI is mainly set.  

But in generally you are right, the Accounts button isn't right.  I was the one that put it there, but if I had to do it all over again I would put a drop down arrow at the top next to the accounts name to switch accounts.  That would look better and feel better to users.
